docs/protocol/substrate-cli.mdx

```diff
@@ -107,91 +108,13 @@ We will be downloading two files for your respective operating system.
-
-  ````yml
-  version: "3.7"
-  services:
-  node:
-    # Replace `snapshot-DATE` with the latest release (like `snapshot-2023-aug-18`)
-    # For running on Aarch64 add `-aarch64` after `DATE`
-    image: ghcr.io/subspace/node:snapshot-DATE
-    volumes:
-  # Instead of specifying volume (which will store data in `/var/lib/docker`), you can
-  # alternatively specify path to the directory where files will be stored, just make
-  # sure everyone is allowed to write there
-      - node-data:/var/subspace:rw
-  #   - /path/to/subspace-node:/var/subspace:rw
-    ports:
-  # If port 30333 or 30433 is already occupied by another Substrate-based node, replace all
-  # occurrences of `30333` or `30433` in this file with another value
-      - "0.0.0.0:30333:30333"
-      - "0.0.0.0:30433:30433"
-    restart: unless-stopped
-    command: [
-      "--chain", "gemini-3f",
-      "--base-path", "/var/subspace",
-      "--execution", "wasm",
-      "--blocks-pruning", "256",
-      "--state-pruning", "archive",
-      "--port", "30333",
-      "--dsn-listen-on", "/ip4/0.0.0.0/tcp/30433",
-      "--rpc-cors", "all",
-      "--rpc-methods", "unsafe",
-      "--rpc-external",
-      "--no-private-ipv4",
-      "--validator",
-    # Replace `INSERT_YOUR_ID` with your node ID (will be shown in telemetry)
-      "--name", "INSERT_YOUR_ID"
-    ]
-    healthcheck:
-      timeout: 5s
-    # If node setup takes longer than expected, you want to increase `interval` and `retries` number.
-      interval: 30s
-      retries: 60
-
-      farmer:
-        depends_on:
-          node:
-            condition: service_healthy
-        # Replace `snapshot-DATE` with latest release (like `snapshot-2023-aug-18`)
-        # For running on Aarch64 add `-aarch64` after `DATE`
-        image: ghcr.io/subspace/farmer:snapshot-DATE
-        volumes:
-    # Instead of specifying volume (which will store data in `/var/lib/docker`), you can
-    # alternatively specify path to the directory where files will be stored, just make
-    # sure everyone is allowed to write there
-        - farmer-data:/var/subspace:rw
-    #   - /path/to/subspace-farmer:/var/subspace:rw
-    ports:
-    # If port 30533 is already occupied by something else, replace all
-    # occurrences of `30533` in this file with another value
-        - "0.0.0.0:30533:30533"
-      restart: unless-stopped
-      command: [
-        "farm",
-        "--node-rpc-url", "ws://node:9944",
-        "--listen-on", "/ip4/0.0.0.0/tcp/30533",
-        # Replace `WALLET_ADDRESS` with your Polkadot.js wallet address
-        "--reward-address", "WALLET_ADDRESS",
-        # Replace `PLOT_SIZE` with plot size in gigabytes or terabytes, for example 100G or 2T (but leave at least 60G of disk space for node and some for OS)
-        "path=/var/subspace,size=PLOT_SIZE",
-      ]
-      volumes:
-        node-data:
-        farmer-data:
-  ```
+  1. Generate `docker-compose.yaml` file:
```

Comment:
Let's make instructions slightly more clear.
1. Input your parameters in the provided form and click `Generate`
2. Copy the generated `docker-compose.yml` file (the generated file should be in this section)

All latter instructions are clear and could be added next.

Reply:
Before the form, please :)